### PR TITLE
K8docsupdate

### PR DESCRIPTION
**Pull request by Le-Zheng**

### CODE REVIEW

File: docker/hyperzoo/README.md [hunk 1]
```diff
@@ -20,6 +20,22 @@ Analytics Zoo hyperzoo image has been built to easily run applications on Kubern
...
+```
+
+Restart dockerd：
+
+```bash
+sudo systemctl restart docker
+```
+
+Then pull the image.
+
```
Comment (glorysdj) [inline docker/hyperzoo/README.md:38]: the commands are different in different os and versions of Docker

**Comment by glorysdj** [comment]
Docker 版本在 1.12 或更高
 /etc/docker/daemon.json 
{
    "registry-mirrors": [
        "加速地址"
    ],
    "insecure-registries": []
}

Docker 版本在 1.8 与 1.11 之间
您可以找到 Docker 配置文件，不同的 Linux 发行版的配置路径不同，具体路径请参考 Docker官方文档，在配置文件中的 DOCKER_OPTS 加入
--registry-mirror=加速地址 
重启Docker，不同的 Linux 发行版的重启命令不一定相同，一般为
service docker restart

Docker for Mac and Docker for Windows

please refer to http://guide.daocloud.io/dcs/docker-9153151.html

File: docker/hyperzoo/README.md (hunk 1 above)
Comment (Le-Zheng) [reply docker/hyperzoo/README.md:42]: updated

File: docs/docs/ProgrammingGuide/k8s.md [hunk 2]
```diff
@@ -19,6 +19,27 @@ Analytics Zoo hyperzoo image has been built to easily run applications on Kubern
 sudo docker pull intelanalytics/hyper-zoo:0.8.0-SNAPSHOT-2.4.3-0.17
 ```
 
+To speed up pulling the image, config docker. If the docker version is higher than 1.12, config the Docker daemon. Edit `/etc/docker/daemon.json` and add the registry-mirrors key and value. Here is an example to use ustc mirror. You can also choose others instead:
+
+```bash
+{
+  "registry-mirrors": ["https://docker.mirrors.ustc.edu.cn/"]
```
Comment (jason-dai) [inline docs/docs/ProgrammingGuide/k8s.md:26]: these mirrors are only for China users, yes? I don't think you should make it as a default example; instead, add a section on how to add mirror for China users.

File: docs/docs/ProgrammingGuide/k8s.md [hunk 3]
```diff
@@ -19,6 +19,27 @@ Analytics Zoo hyperzoo image has been built to easily run applications on Kubern
...
+
+```bash
+sudo systemctl daemon-reload
+sudo systemctl restart docker
+```
+
+If your docker version is between 1.8 and 1.11, find the docker configuration which location depends on the operation system. Edit and add like `DOCKER_OPTS="--registry-mirror=http://hub-mirror.c.163.com"`. Restart docker `sudo service docker restart`.
+
+If you would like to speed up pulling this image on mac or windows, config docker by referencing [here](http://guide.daocloud.io/dcs/docker-9153151.html).
+
```
Comment (jason-dai) [inline docs/docs/ProgrammingGuide/k8s.md:40]: see the comments above

File: docker/hyperzoo/README.md [hunk 4]
```diff
@@ -17,9 +17,30 @@ Analytics Zoo hyperzoo image has been built to easily run applications on Kubern
 - Pull an Analytics Zoo k8s image:
 
 ```bash
-sudo docker pull intelanalytics/hyper-zoo:0.8.0-SNAPSHOT-2.4.3-0.17
+sudo docker pull intelanalytics/hyper-zoo:latest
 ```
```
Comment (jason-dai) [inline docker/hyperzoo/README.md:22]: Add a subsection "Speed up pulling image by adding mirrors", and put the new contents in that subsection. Describe it in a generic fashion, and use mirror in China as an example.

**Comment by jason-dai** [comment]
Also need to update https://analytics-zoo.github.io/master/#DockerUserGuide/

File: docker/hyperzoo/README.md [hunk 5]
```diff
@@ -17,9 +17,30 @@ Analytics Zoo hyperzoo image has been built to easily run applications on Kubern
...
+Flush changes and restart docker：
+
+```bash
+sudo systemctl daemon-reload
+sudo systemctl restart docker
+```
+
+If your docker version is between 1.8 and 1.11, find the docker configuration which location depends on the operation system. Edit and add `DOCKER_OPTS="--registry-mirror=https://<my-docker-mirror-host>"`. Restart docker `sudo service docker restart`.
+
+If you would like to speed up pulling this image on mac or windows, config docker by referencing [here](http://guide.daocloud.io/dcs/docker-9153151.html).
```
Comment (jason-dai) [inline docker/hyperzoo/README.md:40]: Need to use an English version, not a Chinese version

**Comment by Le-Zheng** [comment]
> Also need to update https://analytics-zoo.github.io/master/#DockerUserGuide/

Shall we replace the image with our all-in-one image in this guide? or only add docker hub mirror content?

**Comment by jason-dai** [comment]
> > Also need to update https://analytics-zoo.github.io/master/#DockerUserGuide/
> 
> Shall we replace the image with our all-in-one image in this guide? or only add docker hub mirror content?

I'm fine either way; we need to make sure that the user can follow the descriptions in the guide to complete the tasks.

**Pull request merged**